### DISCUSSION TITLE
Te urządzenie wył. przez wpis konfiguracji

### DISCUSSION THREAD

**Discussion by kendy77** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#119] [General]
I nie zbiera danych :( Nie wiem co sie porobilo, bo nic nie zmienialem ale niestety w integracji mam "Te urządzenie wył. przez wpis konfiguracji" - ktos wie jak to naprawic?

**Comment by PiotrMachowski** [comment]
Wchodzisz w urządzenie i w prawym górnym rogu okna masz ołówek. Jak go klikniesz, to możesz włączyć urządzenie

**Reply by kendy77** [reply]
Dzięki za szybka odpowiedz, ale niestety po wejsciu 'wlacz urzadzenie' nie jest podswietlone i nie moge go wlaczyc :(

**Reply by kendy77** [reply]
tu obrazek:

https://ibb.co/mh9RRzb

**Reply by PiotrMachowski** [reply]
Dziwne... Możesz spróbować usunąć integrację i dodać ponownie

**Reply by krzyzak44** [reply]
@kendy77 udało Ci się odnaleźć przyczynę? Wygląda na to, że mam tak samo u siebie.

**Comment by kendy77** [comment]
tak, robiłem to kilka razy i nic :( Efekt nadal ten sam :(
A 2 dni działało

pt., 31 mar 2023 o 12:13 Piotr Machowski ***@***.***>
napisał(a):

**Reply by PiotrMachowski** [reply]
No to nie wiem, może coś w logach HA się Ci uda znaleźć. Możesz: usunąć integrację, zrestartować HA, dodać integrację
